feature/getting-started-updates (#2504)
mercadopago/devsite-docs: guides/additional-content/getting-started/getting-started.pt.md

O primeiro passo para começar a integrar, seja para você ou para outras pessoas, é **ter uma conta no Mercado Pago** ou no Mercado Livre. Se você não tem uma conta no Mercado Pago, ----[mlb]----[**clique aqui**](https://www.mercadopago.com.br/hub/registration/landing) ------------ ----[mla]----[**clique aqui**](https://www.mercadopago.com.ar/hub/registration/landing) ------------  ----[mlm]----[**clique aqui**](https://www.mercadopago.com.mx/hub/registration/landing) ------------ ----[mpe]----[**clique aqui**](https://www.mercadopago.com.pe/hub/registration/landing) ------------ ----[mco]----[**clique aqui**](https://www.mercadopago.com.co/hub/registration/landing) ------------ ----[mlu]----[**clique aqui**](https://www.mercadopago.com.uy/hub/registration/landing) ------------ ----[mlc]----[**clique aqui**](https://www.mercadopago.cl/hub/registration/landing) ------------ para criar uma gratuitamente.

### Suas integrações

> Se você escolher um produto que requeira integração, deverá acessar [Suas integrações](https://www.mercadopago.com/developers/panel/app), que permite gerenciar integrações com desenvolvimento de código, seus aplicativos, realizar testes e configurar notificações que fornecem informações sobre as transações.

Reviewer: ```suggestion
Se você escolher um produto que requeira integração, deverá acessar [Suas integrações](https://www.mercadopago.com/developers/panel/app), que permite gerenciar integrações com desenvolvimento de código, suas aplicações, realizar testes e configurar notificações que fornecem informações sobre as transações.
```

## Processar pagamentos online

Soluções para quem vende pelo site, WhatsApp ou redes sociais. Oferecemos integração facilitada e acesso gratuito à nossa API. As taxas, baseadas no período de recebimento e/ou na quantidade de parcelas sem juros oferecidas ao comprador, são automaticamente aplicadas às vendas. Para mais detalhes, consulte os links abaixo.

### Soluções sem integração
| Solução | Descrição | Conhecimentos necessários |
|:---|:---|:---|
| [Link de pagamento](https://www.mercadopago[FAKER][URL][DOMAIN]/ferramentas-para-vender) | Crie um link de pagamento em poucas etapas e use na hora para receber pagamentos por chat e redes sociais. Você não precisa de conhecimentos técnicos. | Básico |
| [Planos de assinatura](https://www.mercadopago[FAKER][URL][DOMAIN]/ferramentas-para-vender/suscripciones) | Administra pagamentos periódicos de forma simples e sem a necessidade de realizar uma integração. Permite que os clientes paguem sem a necessidade de um site. | Básico | 

### Soluções com integração

----[mlb]----
| Solução | Descrição | Conhecimentos necessários |
|:---|:---|:---|

> | [Checkout Pro](/developers/pt/docs/checkout-pro/landing) | Processe pagamentos no ambiente do Mercado Pago, de forma segura e simples com um design prontos para usar. Desenvolvemos este checkout otimizado para aumentar a conversão. | Intermediário |

Reviewer: ```suggestion
| [Checkout Pro](/developers/pt/docs/checkout-pro/landing) | Processe pagamentos no ambiente do Mercado Pago, de forma segura e simples com um design pronto para usar. Desenvolvemos este checkout otimizado para aumentar a conversão. | Intermediário |
```

> | [Checkout Transparente](/developers/pt/docs/checkout-api/landing) | Uma solução versátil para que você construa seu processador de pagamentos no seu site. Você controla toda a experiência, de configurações básicas a avançadas. | Avançado|

Reviewer: ```suggestion
| [Checkout Transparente](/developers/pt/docs/checkout-api/landing) | Uma solução versátil para que você construa seu processador de pagamentos no seu site. Você tem o controle total sobre a experiência, desde as configurações básicas até as mais avançadas. | Avançado |
```

| [Checkout Bricks](/developers/pt/docs/checkout-bricks/landing) | Uma solução de pagamento modular, segura e personalizável, onde você controla toda a experiência e escolhe qual Brick e tema usar com base nas necessidades do seu site. | Avançado |
| [Assinaturas](/developers/pt/docs/subscriptions/landing) | Automatiza as cobranças recorrentes através de uma assinatura. Você pode configurar a frequência dos pagamentos e o valor a ser cobrado. | Intermediário |
------------

----[mla, mlm, mpe, mco, mlu, mlc]----
| Solução | Descrição | Conhecimentos necessários |
|:---|:---|:---|

> | [Checkout Pro](/developers/pt/docs/checkout-pro/landing) | Processe pagamentos no ambiente do Mercado Pago, de forma segura e simples com um design prontos para usar. Desenvolvemos este checkout otimizado para aumentar a conversão. | Intermediário |

Reviewer: ```suggestion
| [Checkout Pro](/developers/pt/docs/checkout-pro/landing) | Processe pagamentos no ambiente do Mercado Pago, de forma segura e simples com um design pronto para usar. Desenvolvemos este checkout otimizado para aumentar a conversão. | Intermediário |
```

> | [Checkout API](/developers/pt/docs/checkout-api/landing) | Uma solução versátil para que você construa seu processador de pagamentos no seu site. Você controla toda a experiência, de configurações básicas a avançadas. | Avançado| 

Reviewer: ```suggestion
| [Checkout API](/developers/pt/docs/checkout-api/landing) | Uma solução versátil para que você construa seu processador de pagamentos no seu site. Você tem o controle total sobre a experiência, desde as configurações básicas até as mais avançadas. | Avançado | 
```

| [Checkout Bricks](/developers/pt/docs/checkout-bricks/landing) | Uma solução de pagamento modular, segura e personalizável, onde você controla toda a experiência e escolhe qual Brick e tema usar com base nas necessidades do seu site. | Avançado |
| [Assinaturas](/developers/pt/docs/subscriptions/landing) | Automatiza as cobranças recorrentes através de uma assinatura. Você pode configurar a frequência dos pagamentos e o valor a ser cobrado. | Intermediário |
------------

## Oferecer pagamentos presenciais

Soluções para quem vende de casa, na rua, no carro ou na loja.

----[mla]----
| Solução | Descrição | Conhecimentos necessários |
|:---|:---|:---|
| [Código QR](/developers/pt/docs/qr-code/landing)| Escolha um modelo de integração para gerar códigos QR e fornecer uma experiência completa de pagamento presencial. | Intermediário |
| [Mercado Pago Point](/developers/pt/docs/mp-point/landing) | Integre o Mercado Pago Point e conecte a maquininha de cartão a qualquer sistema iOS ou Android para aceitar pagamentos nas frentes de caixa. | Intermediário |

> | [Mercado Pago Delivery](/developers/pt/docs/mp-delivery/introduction) | Uma solução de entrega de refeições sob demanda, na qual o cliente acessa em uma lista de restaurantes cadastrados, faz seu pedido e paga diretamente pelo app do Mercado Pago. | Intermediário |

Reviewer: ```suggestion
| [Mercado Pago Delivery](/developers/pt/docs/mp-delivery/introduction) | Uma solução de entrega de refeições sob demanda, na qual o cliente acessa uma lista de restaurantes cadastrados, faz seu pedido e paga diretamente pelo app do Mercado Pago. | Intermediário |
```
